extensions/blocks/wordads/edit.js

```diff
@@ -32,12 +32,19 @@ class WordAdsEdit extends Component {
 		const { format, hideMobile } = attributes;
 		const selectedFormatObject = AD_FORMATS.filter( ( { tag } ) => tag === format )[ 0 ];
 		const adControls = (
-			<ToggleControl
-				className="jetpack-wordads__mobile-visibility"
-				checked={ Boolean( hideMobile ) }
-				label={ __( 'Hide ad on mobile views', 'jetpack' ) }
-				onChange={ this.handleHideMobileChange }
-			/>
+			<Fragment>
```

Comment:
Do you think we could do without that Fragment here? It does not seem necessary right now?

Reply:
Great feedback! Updated the PR to remove <Fragment>.